### ISSUE TITLE
Question about stereo upmixing

### ISSUE THREAD

**Issue by NekoAlosama** [ShanonPearce/ASH-Listening-Set#1]
All of the audio content I listen to only uses 2 channels, but I would still like to use all of the BRIR .wav's instead of just the A-30 and A30 .wav's for stereo content. However, this project doesn't have a suggestion for how to upmix stereo for this purpose. I have been able to find an idea from HeSuVi (```Copy: L=0.5*L R=0.5*R C=0.2*L+0.2*R SUB=0.0 RL=0.3*L+-0.2*R RR=-0.2*L+0.3*R SL=0.45*L+-0.25*R SR=-0.25*L+0.45*R```, though I just multiplied all of the values by 2 to result in ```Copy: C=0.4*L+0.4*R SUB=0 RL=0.6*L+-0.4*R RR=-0.4*L+0.6*R SL=0.9*L+-0.5*R SR=-0.5*L+0.9*R```), but I don't know if this makes sense/is valid for this purpose.

Here is what my current configuration looks like:
![Editor_YVvV70lo5J](https://user-images.githubusercontent.com/47763383/193867779-28a402e7-33b4-4af1-aacd-d2fc1f4f81c3.png)

**Comment by NekoAlosama** [comment]
@ShanonPearce

**Comment by NekoAlosama** [comment]
@ShanonPearce 
Definitely seems the values for the upmix requires some experimentation, so I'll look around for anything that claims to be better with good reasoning. The small delay for the rear channels makes sense, though I don't know if the BRIR .wav's already incorporate this. A 4 sample delay also seems to be quite low, to the point that it might not even be noticable (I cannot access my computer at this time), since 4 samples should just be about 0.0000907 seconds worth.

**Comment by NekoAlosama** [comment]
> A 4 sample delay also seems to be quite low, to the point that it might not even be noticable (I cannot access my computer at this time), since 4 samples should just be about 0.0000907 seconds worth.

Nope, it is noticeable. Don't know what would be the best value to put here though.

**Comment by ShanonPearce** [comment]
That's understandable considering some rooms have limited azimuth ranges (rooms 08, 09, 12, 13, 14 & 17) which will limit the surround effect. The BRIR convolution configs for those rooms will just take the largest available azimuth for the rear channels.

**Comment by NekoAlosama** [comment]
@ShanonPearce After thinking about it for a bit, I feel like this issue have been resolved in #2 because a person can just use ASH and HeSuVi with the Upmix Stereo option ticked.
